examples/client/Locomotion/src/services/geo.js

```diff
@@ -113,10 +114,11 @@ const DEFAULT_COORDS = {
     longitude: parseFloat(Config.DEFAULT_LONGITUDE),
   },
 };
-export const getPosition = async () => {
+export const getPosition = async (changeBsPage) => {
   try {
     const granted = await GeoService.checkPermission();
     if (!granted) {
+      changeBsPage(BS_PAGES.LOCATION_REQUEST);
```

Comment:
Change how the function works, it shouldn't change the page, it should return false and then handle it in the component that uses this function